doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
For the case that there is a misunderstanding, in `doc.ddoc` there now is:

> ```
> ROOT_DIR = 
> ROOT = .

And in `std.ddoc`:

> ```
> ROOT_DIR = ../
> ROOT = ..

The `ROOT_DIR` macro is needed to avoid prefixing local URLs with `./`, which is not only ugly, but is also problematic for CHM generation.

The `ROOT` macro is now only used for linking to `index.html` (without actually putting `index.html` in the address bar, which is more ugly than it needs to be).